source/components/JsxParser.tsx

```diff
@@ -329,20 +338,32 @@ export default class JsxParser extends React.Component<TProps> {
 			children = children.props.children
 		}
 
-		return React.createElement(component || lowerName, props, children)
+		const componentProps = {
+			...props,
+			lowerName,
+			component,
+			children,
+		}
+
+		return React.createElement(ForwardRefComponent, componentProps)
+		// return React.createElement(component || lowerName, props, children)
 	}
 
-	render = (): JSX.Element => {
+	render() {
 		const jsx = (this.props.jsx || '').trim().replace(/<!DOCTYPE([^>]*)>/g, '')
 		this.ParsedChildren = this.#parseJSX(jsx)
-		const className = [...new Set(['jsx-parser', ...String(this.props.className).split(' ')])]
-			.filter(Boolean)
-			.join(' ')
+		const className = [...new Set(['jsx-parser', ...String(this.props.className)
+			.split(' ')])].filter(Boolean).join(' ')
```

Comment:
This line is already lamentably hard to read. I'd prefer we either make it readable or leave it alone. Breaking the [ ] onto separate lines makes it even more unclear, IMHO.